docs/internal_api.yml

```diff
@@ -98,3 +139,29 @@ definitions:
     example:
       application/json:
         tenant_id: "1234"
+  UserNew:
+    description: New user descriptor.
+    type: object
+    properties:
+      email:
+        description: User's email.
+        type: string
+        format: email
+      password:
+        description: User's password.
+        type: string
+      propagate:
+        description: |
+          When propagate is true, the useradm will propagate user information
+          to tenantadm, otherwise no request to tenantadm will be made.
+          Defaults to true.
+        type: boolean
+    required:
+      - email
+      - password
+    example:
+      application/json:
+        email: 'user@acme.com'
+        password: 'secret'
+        propagate: false
```

Comment:
I almost don't care, but `propagate` here looks ugly, it doesn't describe the resource but a processing option.

maybe we could move it to the query string so it's: 
`
/tenants/{tenant_id}/users?propagate=true
{
    "username": ...,
    "password": ...
}
`